content/docs/next-release/operator.md

```diff
@@ -391,6 +391,22 @@ storage:
     image: jaegertracing/jaeger-es-index-cleaner  // image of the job
 ```
 
+## Spark dependencies
```

Comment:
Should we change this to `Deriving dependencies`?

Reply:
Based on the other title names it's better to use your proposal.